client/src/components/App.jsx

```diff
@@ -4,6 +4,7 @@ import $ from 'jquery';
 import Content from './Content.jsx';
 import Photo from './Photo.jsx';
 import PhotoModal from './PhotoModal.jsx';
+import '../../dist/styles.css';
```

Comment:
We can rename this to `App.css` now, and follow the same pattern for the other CSS modules

Reply:
And you can keep the css module file in the same directory as the component, if you want. You also don't have to. This is up to you.